docs/customize/vocabularies/users.md

```diff
@@ -71,15 +71,26 @@ current_datastore.activate_user(user)
 db.session.commit()
 ```
 
-## Confirm user
+## Adding users and roles
 
-Only confirmed accounts can be logged in. There are two ways to confirm an account as an administrator:
+You may need to add users or make permission changes after the users vocabulary has been loaded. To create an account programmatically, run:
```

Comment:
minor: let's remove the "programmatically" part in passing. We should reserve it for having to write code (including code in a python shell). 

```suggestion
You may need to add users or make permission changes after the users vocabulary has been loaded. To create an account, run:
```